base/docs/basedocs.jl

```diff
@@ -600,6 +600,16 @@ catch
     @warn "Could not write file."
 end
 ```
+or, when the file cannot be read:
+```julia
+try
```

Comment:
Assign the `try` block to a variable

Reply:
good point

Reply:
Cool. Also can you add blank lines before and after the code blocks?